kafka/kafka-impl/src/main/java/com/datastax/oss/sink/kafka/CassandraSinkTask.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.sink.kafka;
```

Comment:
@pkozhevnikov 
I found that this package name should be com.datastax.oss.kafka.sink and we should drop the class from the "common" module.
Otherwise we are not keeping backward compatibility with existing users of the Kafka Sink

Reply:
the "entry point" of connector is `CassandraSinkConnector`, which is specified in configuration. for the `CassandraSinkTask`, there's no use case of direct usage. (as long as the don't extend these classes)
so, for compatibility, we could make links in old package (empty classes that extend classes from new package) to these two classes.

what do you think?

Reply:
if we do not need the old classes then it is better to drop them.
if it is possible that users have legacy configurations then it is better to use directly the old name or to create a bridge like you suggest (using 'extends').